core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -199,6 +237,12 @@ class R2dbcExecutor(val connectionFactory: ConnectionFactory, log: Logger)(impli
     }
   }
 
+  def updateInBatch(logPrefix: String)(statement: Connection => Statement): Future[Int] =
+    withConnection(logPrefix) { connection =>
+      val boundStmt = statement(connection)
+      updateBatchInTx(boundStmt)
+    }
```

Comment:
Needed a different method here. updateOne is not what we need because it use autoCommit and then we get a different timestamp each time. 

We need a transaction to run the batch.